pycrostates/cluster/_base.py

```diff
@@ -327,8 +327,8 @@ def plot(self, block=False):
 
         Returns
         -------
-        fig : :class:`matplotlib.figure.Figure`
-            Figure
+        fig : Figure
+            Matplotlib figure containing the topographic plots.
```

Comment:
Figure, as for all arguments in `Parameters` / `Returns` should be resolved by `numpydoc` directly.